docs/pages/_data/services/kafka.yml

```diff
@@ -0,0 +1,26 @@
+# Common values:
+
+packageName: beta-kafka
+serviceName: kafka
+techName: Apache Kafka
+
+# Values specific to certain templates:
+
+install:
+  minNodeCount: three
+  nodeDescription: with three brokers
+  serviceAccountInstructionsUrl: https://docs.mesosphere.com/services/kafka/kafka-auth/
+
+managing:
+  podType: kafka
+  taskType: broker
+
+supportedVersions:
+  techExampleVersion: 0.11.0.2
+  techLink: "[Apache Kafka](https://kafka.apache.org/downloads)"
+
+# Values specific to Kafka's own docs content:
+
+kafka:
```

Comment:
What's the value of templating these fields, given that they only apply to this framework?

Reply:
Was imagining that they'd need to be changed on-the-fly as well. It is pretty subjective though.